### PR TITLE
HOTT-4030: Implement Lambda Handler

### PR DESCRIPTION
**Pull request by amberstarlight**
### Jira Link

[HOTT-4030](https://transformuk.atlassian.net/browse/HOTT-4030)

### What?

I have added/removed/altered:

- Added Lambda code to rotate electronic tariff files.

### CODE REVIEW

File: electronic-tariff-file-rotations/main.go [hunk 1]
```diff
@@ -26,3 +40,115 @@ func main() {
...
+				log.Println("Error deleting files.")
+				log.Fatal(err)
+			}
+		}
+
+	} else {
+		log.Printf("No candidates for deletion. Exiting!\n")
+	}
+
+	os.Exit(0)
```
Comment (willfish) [inline electronic-tariff-file-rotations/main.go:153]: I think this is actually implicitly what a go program will do when it exits execution

**Review by willfish — APPROVED** [review]
Looks awesome to me!

I'd probably explicitly handle every error, add some unit tests but and maybe do some jsonl logs just as a habit thing but honestly this is basically good enough IMO!

Is this basically working in the dev env?

File: electronic-tariff-file-rotations/main.go (hunk 1 above)
Comment (amberstarlight) [reply electronic-tariff-file-rotations/main.go:153]: That makes sense. I'll drop this in a commit

**Comment by amberstarlight** [comment]
> Looks awesome to me!
> 
> I'd probably explicitly handle every error, add some unit tests but and maybe do some jsonl logs just as a habit thing but honestly this is basically good enough IMO!
> 
> Is this basically working in the dev env?

JSONL logging is a good idea. Is there a useful package or did you write the logger mostly by hand for the other Lambda(s)?

**Comment by amberstarlight** [comment]
So after our internal discussion - as this isnt live yet I'll come back with unit tests and JSONL logging in further PR(s).

**Pull request merged**